tests/unit/test_utils.py

```diff
@@ -304,6 +304,13 @@ def test_get_temp_filename(self):
                 OSUtils().get_temp_filename(filename)
             )
         )
+    def test_get_temp_filename_len_255(self):
+        filename = 'a'*255
+        self.assertTrue(len(OSUtils().get_temp_filename(filename))<=255)
```

Comment:
Let's use the `assertLessEqual` method instead of `<=`.  It might also make sense to break out the `get_temp_filename` call. Something like:
```python
temp_filename = OSUtils().get_temp_filename(filename)
self.assertLessEqual(len(temp_filename), 255)
```